### PR TITLE
feature: pr-split of layer wise supports of int8

### PR DESCRIPTION
**Pull request by i8run**
## What changes were proposed in this pull request?

This pr is splited from #2759.  It main focus the layer wise supporting, and not fusion relative logics.
Which will make the code clean.

## How was this patch tested?

Jenkins and unit tests.

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Helper.scala [hunk 1]
```diff
@@ -0,0 +1,141 @@
+/*
```
Comment (wzhongyuan) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Helper.scala]: I think other similar class are named `Util` ?

Comment (i8run) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Helper.scala]: Updated. Renamed to `Utils` now.

**Comment by i8run** [comment]
Jenkins passed. http://mlt-bddl01.sh.intel.com:8080/view/PR/job/BigDL-PRVN-lite/458/

**Comment by wzhongyuan** [comment]
LGTM

**Pull request merged**